src/main/java/edu/internet2/middleware/grouper/changeLog/consumer/Office365ApiClient.java

```diff
@@ -390,6 +390,11 @@ public User getUser(Subject subject, String domain) {
         return null;
     }
 
+
+    public User getUser(Subject subject, String domain) {
```

Comment:
looks like this method is not using the domain parameter. Maybe this can be removed.